addons/account/models/account_move.py

```diff
@@ -12,6 +13,9 @@
 from odoo.addons import decimal_precision as dp
 from lxml import etree
 
+
+_logger = logging.getLogger(__name__)
```

Comment:
@nim-odoo don't understand why you have add the logger.

Reply:
@fmdl because [an older version of the change included a logging warning](06b4e6c7fe10d61f0b9fb15ff0bb5d2be356d1fb) as well as the error. The creation of the logger was likely forgotten / missed when the logging itself was removed.